test/test_python.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from rpmlint.checks.PythonCheck import PythonCheck
+from rpmlint.filter import Filter
+
+from Testing import CONFIG, get_tested_package
+
+
+@pytest.fixture(scope='function', autouse=True)
+def pythoncheck():
+    CONFIG.info = True
+    output = Filter(CONFIG)
+    test = PythonCheck(CONFIG, output)
+    return output, test
+
+
+@pytest.mark.parametrize('package', ['binary/pythoncheck-python-doc-in-package'])
+def test_python_doc_in_package(tmpdir, package, pythoncheck):
+    output, test = pythoncheck
+    test.check(get_tested_package(package, tmpdir))
+    out = output.print_results(output.results)
+    assert 'W: python-doc-in-package /usr/lib/python2.7/site-packages/python-mypackage/doc' in out
+    assert 'W: python-doc-in-package /usr/lib/python2.7/site-packages/python-mypackage/docs' in out
+    assert 'W: python-doc-in-package /usr/lib/python3.8/site-packages/python-mypackage/doc' in out
+    assert 'W: python-doc-in-package /usr/lib/python3.8/site-packages/python-mypackage/docs' in out
```

Comment:
Why is this a problem?

Reply:
Waste of space, and you have the %doc macro for this one.
Same applies for the src and tests.

Reply:
Check that it si marked as %doc then instead?

I can imagine situations, where there is a src called submodule. I cannot imagine situations where a src is just a redundant copy of sources -- do you have an example?

Packaging .../site-packages/python-mypackage/tests is totally fine by me. maintainers can decide whether or not to package it into a separate subpackage. At least the warning should not trigger when the package name ends with "test(s)" and it should possibly suggest this when it fins such tests.

Reply:
Standalone documentation (i.e. anything other than inline documentation comments in code) does not belong in a module directory in sitelib, that being reserved for code and possibly data that goes with the code such as rpmlint's `.toml` files. Documentation marked as such with `%doc` does not fit the bill for that, so I'd say this rates a warning.

Reply:
I can imagine a doc submodule that simply handles some doc things. But if this is a warning, I guess I can live with that. Corner cases are everywhere.

Reply:
`/usr/lib64/python3.8/site-packages/numpy/doc` is a real module.

`/usr/lib64/python3.8/site-packages/pygame/docs` is documentation. I'll report that.

Reply:
yet even `/usr/lib64/python3.8/site-packages/pygame/docs` is an importable module.

Reply:
Suggestion: check if `.../site-packages/python-mypackage/docs/__init__.py` exists and only warn if not?

Reply:
Now checking whether it's a module is brilliant, thanks :-)

I'll do that for both doc and src, in fact: if `src` is a Python module I'd give it the benefit of the doubt, but if it isn't it's, quite likely to be source for a component not written in Python that ties into the Python module.

Reply:
Scratch the `src` part, sorry. I'll continue leaving that out entirely. I'm not going to reintroduce that - lost track of what I'd already removed :-)